SKETCH.md

```diff
@@ -0,0 +1,1440 @@
+
+Frequency:
+- More frequently than weekly
+- Weekly
+- Monthly
+- Less frequently than monthly
+- Never but have tried to
+- Never and have never tried to
+
+> **justification**
+>
+> We want to understand the nature of contribution to the PureScript project both
+> to better understand the shape of community involvement and for cohort analysis.
+
+### How often have you felt explicitly welcome in the PureScript community?
+
+Type: matrix
+
+Activities:
+
+- *Official* PureScript community forums or chats (the PureScript Discourse or the PureScript Discord)
+- *Unofficial* PureScript community forums or chats (e.g., reddit.com/r/purescript, Hacker News, the PureScriptPureScript Zulip, etc.)
+- Attending a conference where PureScript is discussed or presented
+- Attending a meetup or local community event where PureScript is discussed or presented
+- Discussion (issues, pull requests, etc.) on a repository *inside* the purescript GitHub organization
+- Discussion (issues, pull requests, etc.) on a repository *outside* of the purescript GitHub organization
+
+Choices:
+
+- I've *often* felt welcome
+- I *sometimes* feel welcome
+- I *never* feel welcome
+- I've never participated in this activity
+
+> **justification**
+>
+> We'd like to know where people are feeling welcome and the degree to which they are feeling welcome.
+
+### How often have you felt *un*welcome in the PureScript community?
+
+Type: matrix (optional)
+
+Activities:
+
+- *Official* PureScript community forums or chats (the PureScript Discourse or the PureScript Discord)
+- *Unofficial* PureScript community forums or chats (e.g., reddit.com/r/purescript, Hacker News, the PureScriptPureScript Zulip, etc.)
+- Attending a conference where PureScript is discussed or presented
+- Attending a meetup or local community event where PureScript is discussed or presented
+- Discussion (issues, pull requests, etc.) on a repository *inside* the purescript GitHub organization
+- Discussion (issues, pull requests, etc.) on a repository *outside* of the purescript GitHub organization
+
+Choices:
+
+- I've *often* felt unwelcome
+- I *sometimes* feel unwelcome
+- I *never* feel unwelcome
+- I've never participated in this activity
+
+> **justification**
+>
+> We'd like to know where people are feeling unwelcome and the degree to which they are feeling unwelcome. This can 
+> help us better understand the free from responses that will come in the next question.
+
+### If you indicated that you did not feel welcome in the PureScript community, are there any details about your experience that you would like to share with us?
+
+Type: free form (optional)
+
+> **justification**
+> 
+> More detail on the type of situations where people have felt unwelcome can let us better 
+> address these issues in the future.
+
+
+### Please let us know your feeling about the following PureScript community working groups.
```

Comment:
No, actually, you're right. This would need a bit more expose. @thomashoneyman , could you recommend a good way to phrase this? We can also cut it out, but I love the idea of working groups as TH presented it and I'd like to include them here to kickstart them if possible.